Flow.Launcher.Core/ExternalPlugins/Environments/AbstractPluginEnvironment.cs

```diff
@@ -80,8 +82,7 @@ internal IEnumerable<PluginPair> Setup()
             }
             else
             {
-                MessageBox.Show(
-                    $"Unable to set {Language} executable path, please try from Flow's settings (scroll down to the bottom).");
+                MessageBox.Show(string.Format(InternationalizationManager.Instance.GetTranslation("runtimePluginUnableToSetExecutablePath"), Language));
```

Comment:
Considering we've implemented MessageboxEx so maybe it's better to use it here.

Reply:
It's implemented in Flow.Launcher project while this file is in Flow.Launcher.Core. I don't know of a way to do that (besides moving MessageBoxEx to a different project) without introducing a circular dependency, and moving MessageBoxEx to a different project would be beyond the scope of this PR.